youtube_dl/extractor/googledrive.py

```diff
@@ -161,12 +164,28 @@ def _get_automatic_captions(self, video_id, subtitles_id, hl):
         return self._get_captions_by_type(
             video_id, subtitles_id, 'automatic_captions', origin_lang_code)
 
+    def _call_api(self, video_id):
+        # Call Google Drive API
+        json_data = self._download_json('https://www.googleapis.com/drive/v3/files/%s?fields=createdTime,modifiedTime,owners&key=%s' % (video_id, self._API_KEY), video_id)
+        return json_data
+
+    # USING URL: https://drive.google.com/file/d/1lVFQrzYKnJDd045Gc9xv1W4YA9zKPX7r/view?usp=sharing
+    # API KEY: AIzaSyCGrlNJSIw19pjonNQOqMIyS2Xai9g0YT0
     def _real_extract(self, url):
         video_id = self._match_id(url)
+        print("_real_extract video_id: ", video_id)
         video_info = compat_parse_qs(self._download_webpage(
             'https://drive.google.com/get_video_info',
             video_id, query={'docid': video_id}))
 
+        json_data = self._call_api(video_id)
+        createdTime = unified_strdate(json_data['createdTime'])
+        modifiedTime = unified_strdate(json_data['modifiedTime'])
+
+        owner_lst = []
```

Comment:
Replace this line and the `for` block with:
```suggestion
        owner_lst = [owner['displayName'] for owner in json_data['owners']
```
*GitHub is too hard to use without a mouse ...*